IHP/Controller/Param.hs

```diff
@@ -261,7 +261,7 @@ instance ParamReader ByteString where
 instance ParamReader Int where
     {-# INLINE readParameter #-}
     readParameter byteString =
```

Comment:
Could add a short testcase to https://github.com/digitallyinduced/ihp/blob/master/Test/Controller/ParamSpec.hs#L26 :)

Reply:
will do